RELEASES.md

```diff
@@ -1,3 +1,122 @@
+Version 1.43.0 (2020-04-23)
+==========================
+
+Language
+--------
+- [Fixed using binary operations with `&{number}` (e.g. `&1.0`) not having
+  the correctly inferred type.][68129]
+
+**Syntax only changes**
+- [Allow `type Foo: Ord` syntactically.][69361]
+- [Unify item parsing & filter illegal item kinds.][69366]
+- [Fuse associated and extern items up to defaultness.][69194]
+- [Permit attributes on `if` expressions.][69201]
```

Comment:
I believe this is a semantic change as well (e.g., you can allow lints as well). cc @Centril

Reply:
Yeah it is.